pubsub/google/cloud/pubsub_v1/subscriber/client.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import pkg_resources
+
+from google.cloud.gapic.pubsub.v1 import subscriber_client
+
+from google.cloud.pubsub_v1 import _gapic
+from google.cloud.pubsub_v1.subscriber.policy import thread
+
+
+__VERSION__ = pkg_resources.get_distribution('google-cloud-pubsub').version
+
+
+@_gapic.add_methods(subscriber_client.SubscriberClient,
+                    blacklist=('pull', 'streaming_pull'))
+class Client(object):
+    """A subscriber client for Google Cloud Pub/Sub.
+
+    This creates an object that is capable of subscribing to messages.
+    Generally, you can instantiate this client with no arguments, and you
+    get sensible defaults.
+
+    Args:
+        flow_control (~.pubsub_v1.types.FlowControl): The flow control
+            settings. Use this to prevent situations where you are
+            inundated with too many messages at once.
+        policy_class (class): A class that describes how to handle
```

Comment:
It may be useful to take a policy instance instead of a class. Depending on the concurrency strategy, the developer may need to pass in initialization options to policy.